webextensions/api/action.json

```diff
@@ -280,6 +280,48 @@
             }
           }
         },
+        "getUserSettings": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/action/getUserSettings",
+            "support": {
+              "chrome": {
+                "version_added": "91"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "116"
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "userSettings_isOnToolbar_property": {
```

Comment:
Is this the usual way of documenting a property on a return value?

Reply:
@queengooborg any advice?

Reply:
This question appeared before at https://github.com/mdn/browser-compat-data/issues/13834 - what's the resolution there?

Reply:
Let's go for `returns_userSettings_isOnToolbar_property` with the description `<code>userSettings.isOnToolbar</code> in returned object`, as that seems to be the consensus for style!

Reply:
> Let's go for `returns_userSettings_isOnToolbar_property` with the description `<code>userSettings.isOnToolbar</code> in returned object`, as that seems to be the consensus for style!

The method is async. Does "returns" apply to async and sync functions?

Reply:
Yeah, that should be fine!